### ISSUE TITLE
There is NOT numerical normalization module in cal_wer.sh, which results in higher WER results.

### ISSUE THREAD

**Issue by LwLiu-2012** [BytedanceSpeech/seed-tts-eval#12]
When I run cal_wer.sh, I found there is NOT numerical normalization module, which results:
Target text: "The primary coil has fifty turns."——> Inference text by Whisper-large: "The primary coil has 50 turns."
"My grandmother has Type One diabetes."——>"My grandmother has type 1 diabetes."

Is the WER result in your SeedTTS paper also obtained in this way?

**Comment by faceless-rex** [comment]
> When I run cal_wer.sh, I found there is NOT numerical normalization module, which results: Target text: "The primary coil has fifty turns."——> Inference text by Whisper-large: "The primary coil has 50 turns." "My grandmother has Type One diabetes."——>"My grandmother has type 1 diabetes."
> 
> Is the WER result in your SeedTTS paper also obtained in this way?

@LwLiu-2012  Yes, the results reported in our paper were obtained exactly with the released code.

**Comment by LwLiu-2012** [comment]
Got it. Thanks for your response.
